lib/create-environment-api.ts

```diff
@@ -159,6 +161,216 @@ export default function createEnvironmentApi(makeRequest: MakeRequest) {
       return wrapAsset(makeRequest, assetData)
     },
 
+    /**
+     *
+     * @description Get a BulkAction by ID.
+     *  See: https://www.contentful.com/developers/docs/references/content-management-api/#/reference/bulk-actions/bulk-action
+     * @param bulkActionId - ID of the BulkAction to fetch
+     * @returns - Promise with the BulkAction
+     *
+     * @example ```javascript
+     * const contentful = require('contentful-management')
+     *
+     * const client = contentful.createClient({
+     *   accessToken: '<content_management_api_key>'
+     * })
+     *
+     * client.getSpace('<space_id>')
+     * .then((space) => space.getEnvironment('<environment-id>'))
+     * .then((environment) => environment.getBulkAction('<bulk_action_id>'))
+     * .then((bulkAction) => console.log(bulkAction))
+     * ```
+     */
+    getBulkAction(bulkActionId: string) {
+      const raw = this.toPlainObject() // Environment object
+
+      return makeRequest({
+        entityType: 'BulkAction',
+        action: 'get',
+        params: {
+          spaceId: raw.sys.space.sys.id,
+          environmentId: raw.sys.id,
+          bulkActionId,
+        },
+      }).then((data) => wrapBulkAction(makeRequest, data))
+    },
+
+    /**
+     * @description Creates a BulkAction that will attempt to publish all items contained in the payload.
+     * See: https://www.contentful.com/developers/docs/references/content-management-api/#/reference/bulk-actions/publish-bulk-action
+     * @param {BulkActionPayload} payload - Object containing the items to be processed in the bulkAction
+     * @returns - Promise with the BulkAction
+     *
+     * @example
+     *
+     * ```javascript
+     * const contentful = require('contentful-management')
+     *
+     * const client = contentful.createClient({
+     *   accessToken: '<content_management_api_key>'
+     * })
+     *
+     * const payload = {
+     *  entities: {
+     *    sys: { type: 'Array' }
+     *    action: 'publish',
```

Comment:
`action` isn't valid on the `publish` payload

Reply:
good 👀